javascript-light-bulb/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./bulb.css" />
+    <title>Light Bulb Switcher</title>
+  </head>
+  <body>
+    <div id="flexing-bulb">
+      <div class="Main_Bulb"></div>
```

Comment:
classes should either be `camelCased` or `kebab-cased`. Classes shouldn't start with a capital letter